src/DragHandle.tsx

```diff
@@ -94,12 +96,18 @@ export default function DragHandle<R, SR>({
     event.stopPropagation();
     const { idx, rowIdx } = selectedPosition;
     const sourceRow = rows[rowIdx];
-    const targetRows = rows.slice(rowIdx + 1);
+    const targetRows: R[] = [];
```

Comment:
I don't think we should update the row when the cell is readonly. This also makes me wonder if we should have the same syntax for `onPaste` and `onFill`. With `onFill` we can update 1 row at a time. We do not have to worry about indexes also. Wdyt?